JSDemos/Demos/Toolbar/Adaptability/description.md

```diff
@@ -1,7 +1,7 @@
-This demo demonstrates multiline mode in the Toolbar component. 
+This demo demonstrates how the Toolbar component can adapt to different screen widths.
 
 To see step-by-step instructions on how to get started with the DevExtreme Toolbar component, refer to the following tutorial: [Getting Started with Toolbar](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/). 
 
-In this demo, you can drag the Toolbar container's resizing handle. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, the Toolbar wraps onto multiple lines. 
+In this demo, you can drag the Toolbar container's resizing handle to see how the component behaves on the width resize. If you enable the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property, the Toolbar wraps onto multiple lines. 
 
 In single-line mode (the [multiline](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/#multiline) property is false), the Toolbar does not wrap. It can display an overflow menu with items that do not fit into the container. Use the [locateInMenu](/Documentation/ApiReference/UI_Components/dxToolbar/Configuration/items/#locateInMenu) property to control whether an item appears in the overflow menu.
```

Comment:
I would start the description with the default behavior (hide items in a context menu) and then move to the multiline mode.